src/content/grant/2025_HDH_Hybrid_Dependency_Hypergraphs.md

```diff
@@ -0,0 +1,14 @@
+---
+name: "HDH: Hybrid Dependency Hypergraphs for Distributing model agnostic quantum workloads"
+year: 2025
+month: 7
+day: 28
+country: UK
+tags:
+  - python
+  - IR
+  - distributed-QC
```

Comment:
@grageragarces feel free to add any other tags you feel are important here (even after this gets merged).